javascript/app-discover/src/components/graph/CausalGraphExplorer.tsx

```diff
@@ -180,27 +177,7 @@ export const CausalGraphExplorer: React.FC<{
 				</Xwrapper>
 			</div>
 			<FlexContainer ref={legendRef}>
```

Comment:
Is this container needed if it's inside a Grid?

Reply:
Yep, this outside container is needed because with it's ref we can get it's height so the graph won't be on top of it